### DISCUSSION TITLE
Distance for some OD pairs is "N/A"

### DISCUSSION THREAD

**Discussion by Anjun93** [jdlph/Path4GMNS#33] [Q&A] [answered]
For the network around ASU, some OD pairs look inaccessible in od_accessibility.csv. The "distance" field and "accessibility" field are "N/A". It's unreasonable.
![image](https://user-images.githubusercontent.com/75136625/199140874-ffdcd802-18d1-403a-b543-8369cb9f14dc.png)

**Comment by Anjun93** [comment]
Here are the network files.
[demand.csv](https://github.com/jdlph/Path4GMNS/files/9906069/demand.csv)
[link.csv](https://github.com/jdlph/Path4GMNS/files/9906070/link.csv)
[node.csv](https://github.com/jdlph/Path4GMNS/files/9906071/node.csv)
[od_accessibility.csv](https://github.com/jdlph/Path4GMNS/files/9906072/od_accessibility.csv)
[zone.csv](https://github.com/jdlph/Path4GMNS/files/9906073/zone.csv)
[zone_accessibility.csv](https://github.com/jdlph/Path4GMNS/files/9906074/zone_accessibility.csv)
[zone_generation.csv](https://github.com/jdlph/Path4GMNS/files/9906075/zone_generation.csv)

**Comment by jdlph** [comment, marked as answer]
Thank you for pointing this out!

The accessibility evaluation is based on the shortest path calculation between each OD pair. When N/A appears in fields _accessibility_ and _distance_ for an OD pair in od_accessibility.csv, it indicates that there is NO path between the two zones or the shortest path exceeds the maximum distance, which is 2147483647 (the maximum positive value for a 32-bit signed binary integer) defined in consts.py.
